packages/nextra/src/client/components/select.tsx

```diff
@@ -71,7 +71,7 @@ export const Select: FC<MenuProps> = ({
               <li
                 className={cn(
                   focus
-                    ? 'x:bg-primary-50 x:text-primary-600 x:dark:bg-primary-500/10'
+                    ? 'x:bg-primary-100 x:text-primary-900 x:dark:text-primary-600 x:dark:bg-primary-500/10'
```

Comment:
```suggestion
                    ? 'x:bg-primary-100 x:text-primary-600 x:dark:bg-primary-500/10'
```
primary-600 looks better on light mode IMO
<img width="249" alt="image" src="https://github.com/user-attachments/assets/3678a324-2587-441d-a3aa-7014a4f4f91f" />

Reply:
the old colors failed wcag

how about now?

![image](https://github.com/user-attachments/assets/88dc50b2-dc94-41c5-b1cf-f7caeabc4b77)

![image](https://github.com/user-attachments/assets/24e79b58-b77c-47c9-82c8-1e9dbde10c71)

Reply:
From your screens lgtm, better than before!